docs/build/messages/read-receipt.mdx

```diff
@@ -158,21 +150,23 @@ Here's how you can receive a read receipt:
 
 ```tsx
 if (message.contentType.sameAs(ContentTypeReadReceipt)) {
-  // We've got a reply.
-  const timestamp = message.content.timestamp;
+  // The message is a read receipt
+  const timestamp = message.sent;
 }
 ```
 
 </TabItem>
 <TabItem value="react" label="React"  attributes={{className: "react_tab"}}>
 
 ```jsx
+import { ContentTypeId } from "@xmtp/react-sdk";
 import { ContentTypeReadReceipt } from "@xmtp/content-type-read-receipt";
 
-if (ContentTypeReadReceipt.sameAs(message.contentType)) {
+const contentType = ContentTypeId.fromString(message.contentType);
+
+if (ContentTypeReadReceipt.sameAs(contentType)) {
   // The message is a read receipt
-  const readReceiptContent = message.content;
-  const timestamp = readReceiptContent.timestamp;
+  const timestamp = message.sentAt;
```

Comment:
While we are here, @rygine - may I ask you about [line 227 below](https://github.com/xmtp/xmtp-dot-org/pull/582/files#diff-4b29f6ae0f31a7446640051703b5cb70c755be7188a60a5ab2743e0811374c8bR227)? 

In particular, I don't understand what this text is saying:

>While iterating through messages, you can be sure that the last message was read at the timestamp of the read receipt if the string of the date is lower.

Is the use of the word "date" correct - or should it be "timestamp"? And what value should it be lower than?

Reply:
i'd say that "timestamp" is a better term as the value is a string that represents a date with hours, minutes, and seconds.